src/main/flight/pid.c

```diff
@@ -172,11 +172,11 @@ float calcHorizonLevelStrength(const pidProfile_t *pidProfile) {
 
 float pidLevel(int axis, const pidProfile_t *pidProfile, const rollAndPitchTrims_t *angleTrim, float currentPidSetpoint) {
     // calculate error angle and limit the angle to the max inclination
-    float errorAngle = pidProfile->levelSensitivity * rcCommand[axis];
+    float errorAngle = pidProfile->levelSensitivity * getRcDeflection(axis);
```

Comment:
rcCommand and getRcDeflection were meant to be different - rcCommand is scaled, expo-ed and otherwise user-alternated value, getRcDeflection was meant to get actual stick deflection(getRcStickDeflection in CF).